src/main/webapp/app/shared/util/download.util.ts

```diff
@@ -31,3 +31,8 @@ export function downloadFile(blob: Blob, filename: string) {
         window.URL.revokeObjectURL(url);
     }
 }
+
+export function downloadStream(data: any, type: string, filename: string) {
```

Comment:
Is it necessary to use `any` or can you specify that further (I'm guessing you can't and need to use the type)?